packages/example-hello-world/src/application.ts

```diff
@@ -0,0 +1,23 @@
+// Copyright IBM Corp. 2018. All Rights Reserved.
+// Node module: @loopback/example-hello-world
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {Application} from '@loopback/core';
+import {RestApplication, RestServer} from '@loopback/rest';
+
+export class HelloWorldApplication extends RestApplication {
```

Comment:
This is different from https://github.com/strongloop/loopback4-example-hello-world, I am leveraging the new `RestApplication` class to avoid configuring the sequence handler inside `start` method (which I consider as too late).